tests/integration/test_inout.py

```diff
@@ -17,7 +17,7 @@ def test_basic(validate):
     def foo(q: qubit) -> None: ...
 
     @guppy(module)
-    def test(q: qubit @owned) -> qubit:
+    def test(q: qubit @ owned) -> qubit:
```

Comment:
it appears my formatter has been at all the decorators. I can undo this if necessary